lib/whenever/numeric_seconds.rb

```diff
@@ -1,17 +1,11 @@
 module Whenever
   class NumericSeconds
-    PATTERN = /(\d+)\.(seconds?|minutes?|hours?|days?|weeks?|months?|years?)/
-
     attr_reader :number
 
     def self.seconds(number, units)
       new(number).send(units)
     end
 
-    def self.process_string(string)
-      string.gsub(PATTERN) { Whenever.seconds($1, $2) }
-    end
-
     def initialize(number)
       @number = number.to_i
```

Comment:
Anyone familiar with this syntax is probably coming from Rails, which patches `Numeric` and allows things like `1.5.days`. I think it'd be best to match that expected behavior here.